lib/random_data.rb

```diff
@@ -0,0 +1,41 @@
+module RandomData
```

Comment:
what is this for? you could use Faker to do this or was the plan to learn to build something like Faker?

Reply:
Yeah...it was a technique they had us use early on and then added faker...good point

Reply:
btw, how would i use faker to pick from a list of values?  the only RandomData method i use is `randome_resource_type` which returns `["tutorial","book","workshop","reference"].sample`

Reply:
I don't think you can. Faker gives you place holders when the meaning of the place holder isn't important. If you want specific ones, you can do that directly in your factory